handlers/handlers.go

```diff
@@ -90,6 +91,23 @@ func handleDefault(r *ranch.Ranch) http.HandlerFunc {
 	}
 }
 
+var (
+	acquireWaitingDurationSeconds = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "boskos_acquire_waiting_duration_seconds",
+		Help:    "Histogram of waiting in seconds for a Boskos' client to acquire a resource.",
+		Buckets: []float64{1, 10, 100, 500, 1000, 1500, 2000},
+	}, []string{
+		"type",
+		"state",
+		"dest",
+		"owner",
```

Comment:
I would lean against including owner here, since this field would have high cardinality.

Reply:
Removed.